doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -1,223 +1,275 @@
 
-Actual result: It is not possible to delete variables and the trash can icon is missing.
+Workaround: You can use the DRL score calculations.
 
-Workaround:
+== {OPENSHIFT}
 
-. Download the process or a case.
-. Locate the case file variable and global variable in the downloaded `.bpmn` file and delete them.
-. Delete the process or a case from {CENTRAL}.
-. Import the previously downloaded and edited `.bpmn` file back into {CENTRAL}.
+ifdef::PAM[]
 
-.Ruleflow group is not populated [https://issues.redhat.com/browse/RHPAM-2740[RHPAM-2740]]
+.You can not create custom database image for PostgreSQL using PostgreSQL enterprise instance [https://issues.redhat.com/browse/RHPAM-3005[RHPAM-3005]]
 
-Issue: In the process designer, the Ruleflow group menu is not populated with the rule flow groups defined in the project.
+Issue: When you create a custom database for PostgreSQL and push the image using PostgreSQL enterprise instance, {KIE_SERVER} fails to start.
 
 Steps to reproduce:
 
-. Create a new DRL file containing a rule flow group.
-. Create a new process.
-. Activate the Business Rule task.
-. Click the *Rule Flow Group* property of the Business Rule task.
+* Create a custom database and make the image build for PostgreSQL.
++
+----
+make build postgresql
+----
+
+* Push the custom database image using PostgreSQL enterprise instance.
++
+----
+db.primary_label=postgresplus116
+db.port=5432
+server_label_primary=postgresplus116
+hibernate.dialect=org.hibernate.dialect.PostgresPlusDialect
+hibernate.connection.url=jdbc\:edb\://edb-as11.mwqe.upshift.rdu2.redhat.com\:5432/dballo06
+hibernate.connection.schema=public
+datasource.class.xa=com.edb.xa.PGXADataSource
+----
 
-Expected result: All the rule flow groups from the project are listed in the *Rule Flow Group* drop-down menu.
+* Deploy the {KIE_SERVER} using S2I.
 
-Actual result: The *Rule flow Group* drop-down menu is empty.
+Expected result: You can create custom database image for PostgreSQL using PostgreSQL enterprise instance.
 
-Workaround:
+Actual result: You can not create custom database image for PostgreSQL using PostgreSQL enterprise instance.
 
-. Click on the canvas.
-. Click the *Rule flow Group* drop-down menu again.
+Workaround: None.
 
 endif::[]
 
-== DMN Designer
+.The {CENTRAL} pod fails to start when the `known_host` file is missing from the githook secret [https://issues.redhat.com/browse/RHPAM-3056[RHPAM-3056]]
 
-.In the DMN Designer, you cannot convert a Java class with that contains an invalid DMN identifier [https://issues.redhat.com/browse/RHDM-1231[RHDM-1231]]
+Issue: The {CENTRAL} pod fails to start and remains stuck at *Container Creating* status if the `known_host` file is missing from the githook secret.
 
-Issue: When you try to convert a Java class to a DMN data type and the Java class contains a field name, you will receive an error.
+Steps to reproduce:
 
-Workaround: None.
+* Create the following post-commit script.
```

Comment:
I'd suggest using a numbered list instead of bullets since these are sequential steps.